CacheManager.StackExchange.Redis.SelfCleaning.Tests/SelfCleaningRedisIntegrationTests.cs

```diff
@@ -0,0 +1,247 @@
+﻿using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Threading.Tasks;
+using CacheManager.Core;
+using CacheManager.Core.Internal;
+using CacheManager.StackExchange.Redis.SelfCleaning.Core;
+using CacheManager.StackExchange.Redis.SelfCleaning.Timers;
+using Moq;
+using NUnit.Framework;
+using StackExchange.Redis;
+
+namespace CacheManager.StackExchange.Redis.SelfCleaning.Tests
```

Comment:
Move this file into "Integrations" folder

Reply:
changed